package.json

```diff
@@ -29,7 +34,7 @@
   },
   "scripts": {
     "test": "mocha",
-    "postinstall": "make",
-    "start": "./bin/slackin $SLACK_SUBDOMAIN $SLACK_API_TOKEN"
+    "postinstall": "./node_modules/.bin/gulp",
+    "start": "node ./bin/slackin"
```

Comment:
I moved this to inside `slackin`:

```
var org = program.args[0] || process.env.SLACK_SUBDOMAIN;
var token = program.args[1] || process.env.SLACK_API_TOKEN;
```

as different shells have ways have getting environment variables. (ie. windows = %SLACK_SUBDOMAIN%)